extensions/vscode/src/multiStepInputs/newConnectCredential.ts

```diff
@@ -508,6 +508,16 @@ export async function newConnectCredential(
     input: MultiStepInput,
     state: MultiStepState,
   ) {
+    if (!state.data.name && typeof state.data.url === "string") {
```

Comment:
Currently `state.data.name` never gets set, but I figured it was good practice to check for this in case another step is created or this step moves for some reason.